ui/app/routes/vault/cluster/access/mfa/index.js

```diff
@@ -1,3 +1,15 @@
 import Route from '@ember/routing/route';
 
-export default class MfaConfigureRoute extends Route {}
+export default class MfaConfigureRoute extends Route {
```

Comment:
adding this here triggers the model to call the mfa-method query method on the mfa-method adapter. This is what checks if there are mfa-methods. Before you had to change the route to trigger this, now whenever the page is refreshed it checks.

Reply:
it seems to work just fine. amended. Thanks for the suggestion.